src/main/java/com/orbitz/consul/util/failover/ConsulFailoverInterceptor.java

```diff
@@ -59,6 +62,7 @@ public Response intercept(Chain chain) throws IOException {
 					// This is because a 400 series error is a valid code (Permission Denied/Key Not Found)
 					return chain.proceed(next);
 				} catch (Exception ex) {
+					LOGGER.debug("Failed to connect to {}", nextRequest.get().url(), ex);
```

Comment:
Added a debug log message here since it is the only place to find the full url and server where the failed request happened. And that's where we realized it was stuck to `http`.
I'll remove it if you want to keep logging to a minimum in this code base.